app/models/pawn.rb

```diff
@@ -13,7 +13,12 @@ def valid_move?(to_x, to_y)
         return false
       end
     end
-    pawn_move_vertical?(to_x, to_y) && vertical_move_only?(to_x, to_y) 
+    pawn_move_vertical?(to_x, to_y) && vertical_move_only?(to_x, to_y)
+  end
+
+  def en_passant(to_x, to_y, capture_pawn)
```

Comment:
We should check for en passant.. probably best to rename to `en_passant?` It should check if a player is in the situation called en passant where she can not make a move without endangering herself..